tool/pmt-builder.js

```diff
@@ -1,23 +1,33 @@
 const mempoolJS = require("@mempool/mempool.js");
 const pmtBuilder = require("../index");
 
+const getPmtInformation = async (network, txHash) => {
+
+    const { bitcoin: { blocks, transactions } } = mempoolJS({
+        hostname: 'mempool.space',
+        network // 'testnet' | 'mainnet'
+    });
+
+    const transaction = await transactions.getTx({ txid: txHash });
+    const blockHash = transaction.status.block_hash;
+
+    const blockTxids = await blocks.getBlockTxids({ hash: blockHash });
+
+    const resultPmt = pmtBuilder.buildPMT(blockTxids, txHash);
+
+    return resultPmt;
+
+};
+
 (async () => {
     try {
         const network = process.argv[2];
-        const hash = process.argv[3];
-        const txHash = process.argv[4];
-
-        const { bitcoin: { blocks } } = mempoolJS({
-            hostname: 'mempool.space',
-            network: network // 'testnet' | 'mainnet'
-        });
-    
-        const blockTxids = await blocks.getBlockTxids({ hash });
-
-        const resultPmt = pmtBuilder.buildPMT(blockTxids, txHash);
-        console.log("Block Transactions Ids:", blockTxids);
-        console.log("Filtered Hash:", txHash);
-        console.log("Result:", resultPmt);
+        const txHash = process.argv[3];
+
+        const pmtInformation = await getPmtInformation(network, txHash);
+
+        console.log('PMT information: ', pmtInformation);
```

Comment:
```suggestion
        const pmt = await getPmt(network, txHash);

        console.log('PM: ', pmt);
```
There no "information" really, right? Just the plain PMT

Reply:
It's really an object like this:

```js
{
  totalTX: 183,
  hashes: [
    '108add0b05ab4c6877c0d4a2500c45ccc52f346cc6429ddd8f24cc63141ff33f',
    'ef659287588630175492aa4edb503bccc68cd1f6ddb0a4592221207da00bfe85',
    '0e499c985808061e72333eb6f24fc21e48776389e9cb2a881502b82d864ea237',
    'cedcdcc461911b79f92a606d73948eeed03a4974692374cb1617e0832ec6319f',
    'e2796c36795f5f30618a897bf80f8dba350c84b673423c3c3e5c207b6d44041c',
    'e50bab6fa2651fc28ca442f58e176f44d344142b5425a27aa38cc80872906a2f',
    'd4cf4a00fbc4ab1dbe591d0c3409cc0708d141db2d344a58f60d94f6e4d8df76',
    'a6bf29b94810f58b2f727d94b9fb0b189bd95ef57c14650eec7dbc0b01b7216f',
    'e703cb8593499a148cda39497b1521f32b9a9e8f3eb76ac96ed0dfc1a7ade2e5'
  ],
  flags: 12518144,
  hex: 'b7000000093ff31f1463cc248fdd9d42c66c342fc5cc450c50a2d4c077684cab050bdd8a1085fe0ba07d20212259a4b0ddf6d18cc6cc3b50db4eaa925417308658879265ef37a24e862db80215882acbe9896377481ec24ff2b63e33721e060858989c490e9f31c62e83e01716cb74236974493ad0ee8e94736d602af9791b9161c4dcdcce1c04446d7b205c3e3c3c4273b6840c35ba8d0ff87b898a61305f5f79366c79e22f6a907208c88ca37aa225542b1444d3446f178ef542a48cc21f65a26fab0be576dfd8e4f6940df6584a342ddb41d10807cc09340c1d59be1dabc4fb004acfd46f21b7010bbc7dec0e65147cf55ed99b180bfbb9947d722f8bf51048b929bfa6e5e2ada7c1dfd06ec96ab73e8f9e9a2bf321157b4939da8c149a499385cb03e703bf0300'
}
```